Common/Interfaces/IBrokerageFactory.cs

```diff
@@ -54,5 +54,11 @@ public interface IBrokerageFactory : IDisposable
         /// <param name="algorithm">The algorithm instance</param>
         /// <returns>A new brokerage instance</returns>
         IBrokerage CreateBrokerage(LiveNodePacket job, IAlgorithm algorithm);
+
+        /// <summary>
+        /// Gets a brokerage message handler
+        /// </summary>
+        IBrokerageMessageHandler CreateBrokerageMessageHandler(IAlgorithm algorithm, AlgorithmNodePacket job,
+             IApi api, TimeSpan? initialDelay = null, TimeSpan? openThreshold = null);
```

Comment:
Do we need all of the parameters on the interface or can some of them be resolved from within the individual implementations